package.json

```diff
@@ -886,6 +891,10 @@
         }
       ],
       "commandPalette": [
+        {
+          "command": "swift.generateLaunchConfigurations",
+          "when": "swift.hasPackage"
```

Comment:
I think this should be based off a new context key called `hasExecutableTarget`. If you create a basic library package (i.e. with `swift package init`) there are no launch configs to generate because there is nothing executable to launch.